srcpkgs/python3-termtosvg/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'python3-termtosvg'
+pkgname=python3-termtosvg
```

Comment:
Should be named termtosvg because this is executable. python3- prefix is for modules that need to be imported.